spec/observe_tail_spec.rb

```diff
@@ -21,6 +21,17 @@
     FileUtils.rm_rf(sincedb_path)
   end
 
+  context "when sincedb path is given but ENV[\"HOME\"] or ENV[\"SINCEDB_PATH\"] is not given" do
+
+    it "should not raise an exception" do
+      _home = ENV.delete("HOME")
```

Comment:
this should be done in a before/after block, even though I understand it's hard to transfer that state over.

The problem with not doing it, is that if this assertion fails, the HOME var is not restored and all subsequent tests will likely fail too.